GridMvc/Html/GridHtmlOptions.cs

```diff
@@ -52,6 +52,11 @@ public IGridHtmlOptions<T> WithPaging(int pageSize, int maxDisplayedItems)
         }
 
         public IGridHtmlOptions<T> WithPaging(int pageSize, int maxDisplayedItems, string queryStringParameterName)
+        {
+            return WithPaging(pageSize, maxDisplayedItems, queryStringParameterName, 0);
+        }
+
+        public IGridHtmlOptions<T> WithPaging(int pageSize, int maxDisplayedItems, string queryStringParameterName, int customItemsCount)
```

Comment:
The parameter `customItemsCount` should be nullable and the default given by the overloads should also be `null` and not `0`.